Dockerfile.bodhi

```diff
@@ -0,0 +1,123 @@
+FROM node:16-alpine as bodhi
```

Comment:
I think if the filename is `bodhi.Dockerfile` then you will have proper syntax highlighting

Reply:
sounds good, changed. BTW for some reason my vscode can auto detect proper syntax highlighting for `Docker.xxx` as well lol